doc/api_callback.rst

```diff
@@ -80,12 +80,16 @@
     `action` value     meaning                                                                           Appearance*
     ================== ================================================================================= ===========
     PKG_CLEANUP        `package` cleanup is being performed.                                             3
-    PKG_DOWNGRADE      `package` is being downgraded.                                                    2
+    PKG_DOWNGRADE      `package` is being installed as a downgrade.                                      2
+    PKG_DOWNGRADED     installed `package` is being downgraded.                                          2
```

Comment:
What about `installed `package` is being removed as a downgrade.`

Reply:
Similar construct could be used for PKG_OBSOLETED, PKG_REINSTALLED, PKG_UPGRADED